packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx

```diff
@@ -381,6 +388,14 @@ export const CallArrangement = (props: CallArrangementProps): JSX.Element => {
                         />
                       </Stack>
                     )}
+                    {
+                      /* @conditional-compile-remove(capabilities) */
+                      props.capabilitiesNotificationBarProps && (
+                        <Stack styles={bannerNotificationStyles}>
+                          <CapabilitiesChangeNotificationBar {...props.capabilitiesNotificationBarProps} />
```

Comment:
Concerned here that the notification tracking is stored within the CapabilitiesChangeNotificationBar. This might mean as we transition from lobby -> call if there was a notification and user dismissed it it might be retriggered? (or lost, depending on the tracking logic).

Could you test what happens if the capabilities change in the lobby and see if the CapabilitiesChangeNotificationBar still shows in the connected state, and as a different test stays dimissed in the CapabilitiesChangeNotificationBar if dismissed?

Reply:
I hoisted a a similar tracking logic to what you did for errorbars but in CallArrangement. Is it better if I also put this logic in callcomposite along with the errorbar tracking? I chose CallArrangement because that is where CapabilitiesNotificationBar is used and where I pass the processed notifications to the CapabilitiesNotificationBar UI component. Also, I would not propagate props all the way from the CallComposite.